packages/Meshfree/test/tstNearestNeighborExodusGenerator.cpp

```diff
@@ -163,13 +163,6 @@ void testUniquelyOwnedProblem(
     using ExecutionSpace = typename Device::execution_space;
     using Scalar = double;
 
-    // Set an epsilon for the search. The meshes (see below) mesh a sphere of
```

Comment:
So we no longer need an epsilon because we now have exact nearest neighbors - right?

Reply:
Correct.